rdev.toml

```diff
@@ -4,6 +4,12 @@
 
 [params]
 
+parallel = true
```

Comment:
I'm concerned about these interacting poorly with CI. Maybe have a user-specific file that is added to .gitignore instead?

Reply:
The default should be parallel.

Also, if ci sets an environment variable, this won't override it.

Cache should also be on by default imo. But I do need to check if ccache is available and not use it if it doesn't exist.